src/rpc/mining.cpp

```diff
@@ -779,8 +779,8 @@ UniValue estimatefee(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "estimatefee nblocks\n"
-            "\nEstimates the approximate fee per kilobyte needed for a transaction to begin\n"
-            "confirmation within nblocks blocks.\n"
+            "\nEstimates the approximate fee per virtual kilobyte needed for a transaction to begin\n"
```

Comment:
nit: I might drop the word virtual here?  I think the last sentence explains sufficiently...